src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -426,100 +432,103 @@ public void acceptRecord(final SAMRecord record) {
+
+                    this.metrics.ON_PROBE_BASES   += onBaitBases;
+                    this.metrics.NEAR_PROBE_BASES += (mappedBases - onBaitBases);
+                }
+                else {
+                    this.metrics.OFF_PROBE_BASES += mappedBases;
                 }
             }
 
-            // Now do the bait overlaps
-            int mappedBases = 0;
-            for (final AlignmentBlock block : rec.getAlignmentBlocks()) mappedBases += TargetMetricsCollector.getNumBasesPassingMinimumBaseQuality(rec, block, minimumBaseQuality);
-            int onBaitBases = 0;
-
-            if (probes != null && !probes.isEmpty()) {
-                for (final Interval bait : probes) {
-                    for (final AlignmentBlock block : rec.getAlignmentBlocks()) {
-                        final int end = CoordMath.getEnd(block.getReferenceStart(), block.getLength());
-                        for (int pos=block.getReferenceStart(), readPos = block.getReadStart();
-                             pos<=end && pos >= bait.getStart() && pos <= bait.getEnd() && minimumBaseQuality <= baseQualities[readPos-1];
-                             ++pos, ++readPos) {
-                            ++onBaitBases;
+            ///////////////////////////////////////////////////////////////////
+            // And lastly, ignore reads falling below the mapq threshold
+            ///////////////////////////////////////////////////////////////////
+            if (this.mapQFilter.filterOut(record)) return;
+
+            // NB: this could modify the record.  See noSideEffects.
+            final SAMRecord rec;
+            if (clipOverlappingReads) {
+                final int numOverlappingBasesToClip = SAMUtils.getNumOverlappingAlignedBasesToClip(record);
+                rec = SAMUtils.clipOverlappingAlignedBases(record, numOverlappingBasesToClip, noSideEffects);
+                metrics.PCT_EXC_OVERLAP += numOverlappingBasesToClip;
+            }
+            else rec = record;
+
+            // Find the target overlaps
+            for (final AlignmentBlock block : rec.getAlignmentBlocks()) {
+                final int length = block.getLength(), refStart = block.getReferenceStart(), readStart = block.getReadStart();
+
+                for (int offset = 0; offset < length; ++offset) {
+                    final int refPos = refStart + offset;
+                    final int readPos = readStart + offset;
+                    final int qual = baseQualities[readPos - 1];
+
+                    if (qual < minimumBaseQuality) {
+                        this.metrics.PCT_EXC_BASEQ++;
+                    }
+                    else {
+                        boolean isOnTarget = false;
+                        for (final Interval target : targets) {
+                            if (refPos >= target.getStart() && refPos<= target.getEnd()) {
+                                isOnTarget = true;
+                                ++this.metrics.ON_TARGET_BASES;
```

Comment:
Yes - we would be.  But that's always been the case, and to prevent this the targets are `uniqued` at the start of operations.  It's hard to find these days given all the various classes involved, but I tracked it down to `TargetMetricsCollector:268-281`.